README.md

```diff
@@ -100,6 +115,12 @@ const svgToImg = require("svg-to-img");
 
 The method returns a svg instance based on the given argument.
 
+### svgToImg.connect(options)
+- `options` <[Object]> Pupetteer [connect options](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#puppeteerconnectoptions) object which will typically contain a `browserWSEndpoint` property.
+- returns: <[SvgToImg]> an instance which is connected with the specified Puppeteer endpoint.
+
+Use this method if you want to connect to a running Puppeteer endpoint (see [here](https://github.com/etienne-martin/svg-to-img/issues/9)). Note that the actual connection is established lazily; this means this method will always return, even when the given `browserWSEndpoint` cannot be reached -- in this case, errors will be thrown later when calling one of the `to` functions.
```

Comment:
I think this portion of the doc is not up-to-date with the actual behavior. 

> errors will be thrown later when calling one of the `to` functions

Looking at the tests, it seams the `connect` method will throw if it can't connect to the websocket endpoint.

https://github.com/etienne-martin/svg-to-img/pull/19/files#diff-4ac74674fef9fbe68755b3bdcb9fbfd4R287

Reply:
Actually, the test case was not quite accurate (connect should be outside of `try`). I updated the test code to reflect this. The readme is actually correct.